src/otx/algo/detection/heads/base_head.py

```diff
@@ -666,9 +666,9 @@ def _export_by_feat_single(
         return multiclass_nms(
             bboxes,
             scores,
-            cfg.max_per_img,
+            max_output_boxes_per_class=200,
```

Comment:
Could you explain why this value should be static?

Reply:
`cfg.max_per_img`
https://github.com/openvinotoolkit/training_extensions/blob/ff2cf14131184f2518146876ace2c990e0e81971/src/otx/algo/detection/mmconfigs/ssd_mobilenetv2.yaml#L22

`max_output_boxes_per_class`
https://github.com/openvinotoolkit/training_extensions/blob/ff2cf14131184f2518146876ace2c990e0e81971/src/otx/algo/detection/mmdeploy/base_detection.py#L44

Both seem different, doesn't it?
The former is from `mmconfigs` and the latter is from `deploy_cfg`.
https://github.com/open-mmlab/mmdeploy/blob/v1.3.1/mmdeploy/codebase/mmdet/models/dense_heads/base_dense_head.py#L191-L206

Reply:
In this case we should follow value of model config. Since the trained model use value from model config.

Reply:
https://github.com/openvinotoolkit/training_extensions/pull/3407/commits/2bd356dbbfb8a706ce524745912d901197b3b9a0